pinot-common/src/main/java/org/apache/pinot/common/function/DateTimeFunctions.java

```diff
@@ -216,4 +217,18 @@ static String toDateTime(Long millis, String pattern) {
   static Long fromDateTime(String dateTimeString, String pattern) {
     return DateTimePatternHandler.parseDateTimeStringToEpochMillis(dateTimeString, pattern);
   }
+
+  /**
+   * Return current time as epoch millis
+   */
+  static Long now() {
+    return System.currentTimeMillis();
+  }
+
+  /**
+   * Return epoch millis value based on a given date time string and it's corresponding format.
+   */
+  public static Long formatDatetime(String input, String format) {
```

Comment:
this function is the same as `fromDateTime` on line 217. Do we need both?

Reply:
will delete